src/components/organisms/EarthEditor/PropertyPane/hooks-queries.ts

```diff
@@ -144,14 +140,14 @@ export default ({
     }
 
     if (mode === "widget") {
-      if (!selectedWidgetId) return undefined;
-      const w = scene?.widgets.find(
-        w =>
-          w.pluginId === selectedWidgetId.pluginId &&
-          w.extensionId === selectedWidgetId.extensionId,
-      );
+      if (selected?.type !== "widget") return;
+      const w = selected.widgetId
+        ? scene?.widgets.find(w => selected.widgetId === w.id)
+        : undefined;
       return {
-        id: selectedWidgetId.pluginId + "/" + selectedWidgetId.extensionId,
+        id: selected.widgetId || `${selected.pluginId}/${selected.widgetId}`,
```

Comment:
```suggestion
        id: selected.widgetId || `${selected.pluginId}/${selected.extensionId}`,
```
This is a mistake, right?

Reply:
Thanks!